src/uct/ib/base/ib_device.h

```diff
@@ -73,20 +81,28 @@ enum {
  * Flags which specify which address fields are present
  */
 enum {
-    UCT_IB_ADDRESS_FLAG_LID      = UCS_BIT(0),
-    UCT_IB_ADDRESS_FLAG_IF_ID    = UCS_BIT(1),
-    UCT_IB_ADDRESS_FLAG_SUBNET16 = UCS_BIT(2),
-    UCT_IB_ADDRESS_FLAG_SUBNET64 = UCS_BIT(3),
-    UCT_IB_ADDRESS_FLAG_GID  = UCS_BIT(4),
-    UCT_IB_ADDRESS_FLAG_LINK_LAYER_IB = UCS_BIT(5),
-    UCT_IB_ADDRESS_FLAG_LINK_LAYER_ETH = UCS_BIT(6)
+    /* If set - ETH link layer, else- IB link layer */
+    UCT_IB_ADDRESS_FLAG_LINK_LAYER_ETH = UCS_BIT(0),
+    /* Used for ETH link layer */
+    UCT_IB_ADDRESS_FLAG_ROCE_IPV6      = UCS_BIT(1),
+    /* Used for IB link layer */
+    UCT_IB_ADDRESS_FLAG_IF_ID          = UCT_IB_ADDRESS_FLAG_ROCE_IPV6,
```

Comment:
i think at this point to ease debug we don't have to reuse the flag since there are enough bits for roce version
we can consider this in the future